### DISCUSSION TITLE
FEPX simulation for ceramics

### DISCUSSION THREAD

**Discussion by 18nanma** [neperfepx/FEPX#119] [Q&A] [answered]
Hi There,
I would like to simulate a ceramic-alumina in FEPX. I generated a model on NEPER but have a tough time simulating on FEPX.
the properties to input are not found in literature such as resolved shear stress and shearing rate and rate sensitivity exponent.
Thanks in advance, any help wou;d be great

**Comment by mkasemer** [comment, marked as answer]
@18nanma: the short answer is: I have no idea.

The long(er) answer is: you have to iterate on parameters _within whatever you assume are reasonable bounds_ until the modeled behavior matches the experimentally observed behavior. This is often done by comparing macroscopic responses between simulation and experiment, with bounds of parameters informed by literature.

Without literature values of the material you are interested in or a reasonable surrogate material, you are stuck either making big assumptions, or you have to go gather the data yourself (the latter being preferable).

**Comment by rquey** [comment]
@18nanma The parameters you mention control _plastic_ deformation (of metals). Does you material deform plastically? If so, it is actually what you want to model?
